pkg/kubelet/sysctl/runtime.go

```diff
@@ -70,16 +72,22 @@ func NewRuntimeAdmitHandler(runtime container.Runtime) (*runtimeAdmitHandler, er
 				Message: "Docker before 1.12 does not support sysctls",
 			},
 		}, nil
+	case rktTypeName:
+		return &runtimeAdmitHandler{
+			result: lifecycle.PodAdmitResult{
+				Admit:   false,
+				Reason:  UnsupportedReason,
+				Message: "Rkt does not support sysctls",
+			},
+		}, nil
+	default:
+		// Return admit for other runtimes.
+		return &runtimeAdmitHandler{
+			result: lifecycle.PodAdmitResult{
```

Comment:
@sttts returning true (what this PR did) has the result of enabling the sysctls portion of the e2e node tests.  I checked CRI-O and it looks good. It's also working on cri-containerd.